persistence-modules/spring-jpa-2/src/main/java/com/baeldung/multitenant/config/TenantContext.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.multitenant.config;
+
+public class TenantContext {
+
+    private static final ThreadLocal<Object> currentTenant = new ThreadLocal<>();
```

Comment:
`private static final` should have uppercase name - `CURRENT_TENANT`

Reply:
Done.